test/manual/x509_auth.test.ts

```diff
@@ -59,7 +59,10 @@ describe('x509 Authentication', function () {
     function () {
       it('fails to authenticate', async function () {
         client = new MongoClient(connectionString.toString(), validOptions);
-        const error = await client.connect().catch(e => e);
+        const error = await client.connect().then(
+          () => null,
+          e => e
```

Comment:
This failed, I hope, flakily. 

This currently prints out the entire MongoClient, which is unnecessary and doesn't help much. Happy to revert.

Reply:
it was flaky